src/sky/pro/java/course1/Employee.java

```diff
@@ -1,4 +1,64 @@
 package sky.pro.java.course1;
 
 public class Employee {
+
+    private final String nameOfEmployee;
+    private int workDepartment;
+    private int salaryOfEmployee;
+    public static int hiredEmployeeCounter = 0;
+    private final int id;
```

Comment:
статические поля лучше группировать отдельно от обычных для лучшей читаемости.
`hiredEmployeeCounter` лучше сделать приватным, поскольку доступ к нему обеспечивается через геттер.
```suggestion
    private static int hiredEmployeeCounter = 0;
    
    private final String nameOfEmployee;
    private int workDepartment;
    private int salaryOfEmployee;
    private final int id;
```